server/package.json

```diff
@@ -4,7 +4,7 @@
     "description": "",
     "main": "index.js",
     "scripts": {
-        "dev": "nodemon -w src src/index.js",
+        "start": "nodemon -w src src/index.js",
```

Comment:
Never use nodemon in production mode use node instead and use nodemon in dev mode only